src/EFCore.PG/Storage/Internal/Mapping/NpgsqlArrayTypeMapping.cs

```diff
@@ -39,32 +40,39 @@ namespace Npgsql.EntityFrameworkCore.PostgreSQL.Storage.Internal.Mapping
     /// </remarks>
     public class NpgsqlArrayTypeMapping : RelationalTypeMapping
     {
-        public RelationalTypeMapping ElementMapping { get; }
+        /// <summary>
+        /// The CLR type of the array items.
```

Comment:
Let's avoid xmldocs on private members and methods unless their purpose really is unclear. In this case the comment is actually wrong.

Reply:
FYI this is reverting to a public property per the earlier review. 

As for the comment being wrong, I actually think that demonstrates this property isn't very clear (at least not to me).... How would you describe this one?

Reply:
Well, EF Core has the concept of a type mapping, which describes a relationship between a CLR type and a database type (described as a string for relational databases). So it's definitely not just a CLR type - if I had to, I'd describe this field as "The type mapping for the array element's type". Note that the mapping also has some other functionalities, such as rendering a literal in SQL and comparing values of the CLR type to know if a property has changed.

In any case, this is a central concept in EF Core itself, and of course not specific to Npgsql.